packages/web/pages/api/alliance.ts

```diff
@@ -0,0 +1,16 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import getAllies from 'server/Alliance'
+import respondToError from 'server/respondToError'
+
+export default async function(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    if (req.method === 'GET') {
```

Comment:
I don't know Next.JS well but seems like there should be a way to specify a `get` endpoint so you don't have to code in this 405 handling manually?

Reply:
funnily enough this is pretty much what the docs show. 

If it was me id have it so that the handler for each method was a named export ie `export function get()`  `export function patch()`  `export function post()`

Reply:
The second example of 
https://nextjs.org/docs/api-routes/introduction 

The first doesnt have it but it sounds like it will respond to just any method.